### PR TITLE
Addresses #3533 | added image example for background-clip

### PR DESCRIPTION
**Pull request by arun-sagar-s-a**
Add media example for background-clip

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [ ] I have performed a self-review of my own writing and code.
- [ ] I have checked my entry and corrected any misspellings.
- [ ] I have made corresponding changes to the documentation if needed.
- [ ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by arun-sagar-s-a** [comment]
@yoiang

File: content/css/concepts/background/terms/background-clip/background-clip.md
Comment (caupolicandiaz) [inline content/css/concepts/background/terms/background-clip/background-clip.md:79]: It looks like the image does not exactly match the properties of the examples. The image below uses a double border, but the examples specify a single dashed line. Also the numbers do not match the examples, content-box should be number 1. Please update the image to match

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arun-sagar-s-a I've added my initial review. Please update the entry, and tag me when you've made the changes. Thanks

**Comment by arun-sagar-s-a** [comment]
@caupolicandiaz  Done. Let me know if there's any other changes needed. I made slight changes to make it easy to look at and understand.

**Comment by arun-sagar-s-a** [comment]
@yoiang Can I get a second review please.

**Review by SSwiniarski — APPROVED** [review]
@arun-sagar-s-a Looks good to me.

**Pull request merged**
